{{cookiecutter.project_dirname}}/docker/local.Dockerfile

```
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-FROM node:16-bullseye-slim
+FROM node:18-bullseye-slim
```

here we can use debian `bookworm`
```suggestion
FROM node:18-bookworm-slim
```